test-suite/misc/4722.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+set -e
+
+# create test files
+mkdir -p misc/4722
+ln -sfT toto misc/4722/tata
```

Comment:
Sorry, it seems `ln -T` doesn't exist on OSX.
It shouldn't be necessary in this case (tata should never be a directory) so just remove it.

Reply:
Fixed.